src/condor_contrib/docker/README.md

```diff
@@ -0,0 +1,145 @@
+HTCondor Containers
+===================
+
+Using the Execute Node Container
+--------------------------------
+
+### Overview
+
+The execute node container can connect to an existing HTCondor pool
+via token authentication (recommended) or pool
+password.  The container uses CCB so exposing ports is not necessary.
```

Comment:
I would further clarify this by saying "because of CCB, this container only needs outbound connectivity to the central manager and submit machines even to run jobs and condor_ssh_to_job.